{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -305,9 +306,9 @@
     # Default file storage is private
     PRIVATE_MEDIAFILES_LOCATION = AWS_LOCATION + "/media"
     DEFAULT_FILE_STORAGE = "{{ cookiecutter.project_slug }}.utils.storages.PrivateMediaStorage"
-    STATICFILES_STORAGE = "{{ cookiecutter.project_slug }}.utils.storages.StaticRootS3Boto3Storage"
+    # STATICFILES_STORAGE = "{{ cookiecutter.project_slug }}.utils.storages.StaticRootS3Boto3Storage"
     COLLECTFAST_STRATEGY = "collectfast.strategies.boto3.Boto3Strategy"
-    STATIC_URL = f"https://{aws_s3_domain}/static/"
+    # STATIC_URL = f"https://{aws_s3_domain}/static/"
     MEDIA_URL = f"https://{aws_s3_domain}/media/"
```

Comment:
This PR is marked Ready For Review. Reason for this code being commented out? If unneeded delete(?) Or was this a temporary change while you were testing?

Reply:
Well I left it in there because should we want to return and re-add static files to work through a cdn this line would be needed again, my only purpose in this PR was to fix the bug that was in the code and not letting us use AWS storage. Can remove if you think it is necessary

Reply:
Not a fan of commented out code as it has a habit of never changing. But I think it's fine for now.